package.json

```diff
@@ -24,7 +24,9 @@
     "retry": "git-cz --retry",
     "test": "karma start ./test/karma.conf.js --single-run && jest",
     "tdd:karma": "karma start ./test/karma.conf.js --auto-watch",
-    "tdd:jest": "jest --watch"
+    "tdd:jest": "jest --watch",
+    "storybook": "start-storybook -p 6006 -c ./component-library/.storybook",
+    "build-storybook": "build-storybook"
```

Comment:
can you add the -c here as well? that way storybook doesn't complain about not finding the configg